src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -700,8 +700,8 @@ public static function debug_data() {
 			$php_version_debug .= ' 64bit';
 		}
 
-		if ( function_exists( 'php_sapi_name' ) ) {
-			$php_sapi = php_sapi_name();
+		if ( defined( 'PHP_SAPI' ) ) {
+			$php_sapi = PHP_SAPI;
```

Comment:
Is the `defined()` check needed ? AFAICS, both the function as well as the constant are available as part of PHP Core and cannot be disabled at an extension level.

I can understand that for the function use a call to `function_exists()` is used as hosts may have added the `php_sapi_name` function to the `disabled_functions` ini setting, but AFAIK, there is no such setting for constants, so the constant should be available ?

What am I overlooking ?

Reply:
You are right the constant should always be defined, so we can remove this check altogether.

Reply:
Definitely always defined in PHP, FYI: https://github.com/php/php-src/blob/82aa4253f17777764437a7cd601810368b43ac90/main/main.c#L2120